src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.PowMod.cs

```diff
@@ -3,17 +3,16 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Diagnostics;
-using System.Security;
 
 namespace System.Numerics
 {
+    /// <summary>
+    // Executes different exponentiation algorithms, which are
+    // basically based on the classic square-and-multiply method.
+    // https://en.wikipedia.org/wiki/Exponentiation_by_squaring
+    /// </summary>
```

Comment:
This comment was about the Pow method, not about the BigIntegerCalculator class.